gcloud/datastore/test_client.py

```diff
@@ -18,29 +18,44 @@
 class TestClient(unittest2.TestCase):
 
     DATASET_ID = 'DATASET'
+    CONNECTION = object()
```

Comment:
I'm slightly less favorable to such usage than just `None`:  here, the fact that `None` has a specific meaning to the underlying constructor (particularly that it triggers the implicit majyk) tips the balance.